File: src/test/samples/invalidSamples/invalidSample.m
//#region FirstRegion
#import <Foundation/Foundation.h>
//#endregion

// #endregion Invalid end boundary
// #region Invalid start boundary

//  #region Second Region
@interface MyClass : NSObject
//  #region    InnerRegion  
- (void)myMethod;
//  #endregion  ends InnerRegion   

//  #region
@property (nonatomic, strong) NSString *name;
// #endregion
@end
// #endregion

File: src/test/samples/invalidSamples/invalidSample.swift
import Foundation

//#region FirstRegion
let x = 42
//#endregion

// #endregion Invalid end boundary
// #region Invalid start boundary

//  #region Second Region
class MyClass {
    // #region     InnerRegion
    func myMethod() {}
    //   #endregion   ends InnerRegion

    // #region     
    func myMethod2() {}
    // #endregion   
}
//   #endregion
